fn_grpc_interface/fn_grpc_interface/util/grpc_helper.py

```diff
@@ -0,0 +1,165 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+# -*- coding: utf-8 -*-
+import os
+import sys
+import six
+import logging
+import inspect
+import grpc
+
+try:
+    # PY3
+    import importlib.util
+except ImportError as e:
+    # PY2
+    logging.debug("The Import Error raised for the module importlib trying with imp module : {}".format(e))
+    import imp
+
+
+class GrpcHelperClass(object):
+    def __init__(self, logger_object):
+        self.log_object = logger_object
+
+    def get_grpc_interface_module(self, interface_files=[], base_dir=None, files_dir=None):
```

Comment:
`files_dir` is confusing as you are actually passing the `grpc_package_name` here. Change it to `grpc_package_name` and change the comments too and this will make more sense

Reply:
ok